nc_time_axis/__init__.py

```diff
@@ -2,14 +2,124 @@
 Support for cftime axis in matplotlib.
 
 """
+
+# nc-time-axis provides datetime locator and formatter objects which are
```

Comment:
@trexfeathers in some projects, in the event that they have copied and/or adapted code from other projects, I have seen it be standard practice to include a copy of those projects' licenses in comments at the top of modules.   For example in xarray this is done in the [CFTimeIndex module](https://github.com/pydata/xarray/blob/facafac359c39c3e940391a3829869b4a3df5d70/xarray/coding/cftimeindex.py#L2-L40) as well as other "compatibility" related modules, like [`npcompat.py`](https://github.com/pydata/xarray/blob/main/xarray/core/npcompat.py) or [`pdcompat.py`](https://github.com/pydata/xarray/blob/main/xarray/core/pdcompat.py).  Based on that pattern it seemed natural to do it here in the case of matplotlib, though I'm happy to take direction from the SciTools organization on this practice.